lib/file_helper.rb

```diff
@@ -64,6 +70,7 @@ def self.download(
         verbose: verbose,
         validate_uri: validate_uri,
         timeout: read_timeout,
+        use_port_for_host_header: use_port_for_host_header,
```

Comment:
Nitpick: I wonder if this should be `include_port_in_host_header`, otherwise it sounds like we're replacing the host header with just the port?

Reply:
Ah yes much better will change :+1: